features/topology-map/plugins/org.opennms.features.topology.plugins.topo.vmware/src/main/java/org/opennms/features/topology/plugins/topo/vmware/internal/VmwareTopologyProvider.java

```diff
@@ -318,4 +329,29 @@ private void addVirtualMachine(OnmsNode virtualMachine) {
                 graph.getVertex(getNamespace(), vmwareManagementServer + "/" + vmwareHostSystemId)
         );
     }
+
```

Comment:
What's the difference between these methods and the ones in `VmwareImporter`?

Reply:
If it is a duplication, it may be unified.

Reply:
Yes maybe, but not in the scope of this PR.